Split README.md into smaller, easier-to-read pieces (#1520)
StreisandEffect/streisand: README.md

The Internet can be a little unfair. It's way too easy for ISPs, telecoms, politicians, and corporations to block access to the sites and information that you care about. But breaking through these restrictions is *tough*. Or is it?

If you have an account with a cloud computing provider, Streisand can set up a new node with many censorship-resistant VPN services nearly automatically. You'll need a little experience with a Unix command-line. (But without Streisand, it could take days for a skilled Unix administrator to configure these services securely!) At the end, you'll have a private website with software and instructions.

Here's what **[a sample Streisand server](http://streisandeffect-demo.s3-website.us-east-2.amazonaws.com/)** looks like.

There's a [list of supported cloud providers](#cloud-providers); experts may be able to use Streisand to install on many other cloud providers.

## VPN services

> People usually refer to tools to avoid network censorship as Virtual Private Networks (VPNs). 

Reviewer: ```suggestion
One tool people use to avoid network censorship are Virtual Private Networks (VPNs). 
```

    * This protocol is widely used by multi-national corporations, and might not be blocked.
* [OpenVPN](https://openvpn.net/index.php/open-source.html)
    * [Stunnel](https://www.stunnel.org/index.html) add-on available.
* [Shadowsocks](https://shadowsocks.org/en/index.html), 
    * [simple-obfs](https://github.com/shadowsocks/simple-obfs) add-on available.
* A private [Tor](https://www.torproject.org/) bridge relay
    * [Obfsproxy](https://www.torproject.org/projects/obfsproxy.html.en) with obfs4 available as an add-on.
* [WireGuard](https://www.wireguard.com/), a modern high-performance protocol.

See also:
* A [more technical list of features](Features.md) 
* A [more technical list of services](Services.md)

<a id="cloud-providers"></a>
## Cloud providers
* Amazon Web Services (AWS)
* Microsoft Azure
* Digital Ocean
* Google Compute Engine (GCE)
* Linode
* Rackspace

#### Other providers
If you are an expert and can set up a *fresh Ubuntu 16.04 server* elsewhere, Streisand includes code for:

* Using `ssh` to connect to your remote server from your local machine

> * After connecting to your remote server, you can run Streisand *in place*, called "localhost deployment". This converts the server to a Streisand node.

Reviewer: I think this bullet and the prev could be condensed into one like:

* Using `ssh` to connect to a remote server to run the Streisand builder *in place*, converting the remote server into a Streisand VPN server. We call this "localhost deployment".

I would avoid the term "Streisand node" since I don't think its used elsewhere in the README to this point.

Author: This is split up because of "existing server" vs "localhost" deployment. I've rewritten.

Author: OK, I took your advice and just punted everybody to the advanced instructions.

> "Localhost deployment" often works better than `ssh` deployment. 

Reviewer: I think this comment is missing some context (otherwise why isn't it the default method the README recommends to everyone?).

Adding the context will probably make this too lengthy so I'd recommend just striking this sentence and letting advanced users that need it find out about it via the other pages of docs you link out to.

For more information, see [the advanced installation instructions](Advanced%20installation.md).

## Installation

> You need command-line access to a Unix system. You can use Linux, BSD, or macOS; on Windows 10, the Windows Subsystem for Linux (WSL) counts as Linux. 

Reviewer: Earlier in the README Windows is called out as unsupported and WSL is said to maybe work. This part of the README makes it sound like it will probably work and is supported.

Author: I think I resolved the opinions on this.

Once you're ready, see the [full installation instructions](Installation.md).

> Upcoming Features

Reviewer: I would vote to remove this whole section. At best its optimistic and at worst its inaccurate :laughing: Either way it probably doesn't belong in the top level README as opposed to somewhere else.

Author: I changed to "Things we want to do better"; I'd like to present ourselves as imperfect and looking for help.

-----------------
* Easier setup.
* Faster adoption of new censorship-avoidance tools

We're looking for help with both.

> If there is something that you think Streisand should do, or if you find a bug in its documentation or execution, please file a report on the [Issue Tracker](https://github.com/StreisandEffect/streisand/issues). We appreciate 

Reviewer: typo: This ends abruptly with just "We appreciate"

Author: Fixed.
